searchlib/src/vespa/searchlib/tensor/hnsw_index_saver.cpp

```diff
@@ -6,7 +6,31 @@
 
 namespace search::tensor {
 
-HnswIndexSaver::~HnswIndexSaver() {}
+namespace {
+
+size_t
+countValidLinks(const HnswGraph & graph) {
```

Comment:
Consider calling the function `count_valid_link_arrays` since it counts the number of valid number of link arrays and 
not the valid number of links.

Reply:
Fixed